app/controllers/api/status_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Api::StatusController < Api::ApplicationController
   before_action :require_api_key
+  before_action :require_internal_api_key
```

Comment:
The endpoint itself can't require an internal api key.... we want to just increase what's returned when there is an internal api key used

Reply:
Hmm ok. I'll need to figure out how to split this out.

Reply:
Updated the response to use serializers instead of `as_json`. This should be returning all of the same fields and optionally `score` and `repository_maintenance_stats`. I need to do some more testing around those two to make sure they are working completely correctly.